packages/legend-application-query/src/application/LegendQueryApplicationConfig.ts

```diff
@@ -16,14 +16,26 @@
 
 import {
   assertNonNullable,
+  assertTrue,
   guaranteeNonEmptyString,
+  guaranteeNonNullable,
 } from '@finos/legend-shared';
 import {
   LegendApplicationConfig,
   type LegendApplicationConfigurationInput,
   type LegendApplicationConfigurationData,
 } from '@finos/legend-application';
 
+type SDLCConfig = {
+  default: string;
+  instances: {
+    url: string;
+    key: string;
+    label?: string;
+    studioUrl: string;
```

Comment:
why is there a `studioUrl` in `SDLC` config? should we rename this to `ServiceProdutionizationConfig` or this field to `correspondingStudioInstanceUrl`. Also, we could remove `studioUrl` from `query config` and use this one.

Also, let's remove `default` field and do it like this since we have been it for other places (i.e. taxonomy tree config)

```
  default: string;
  instances: {
    url: string;
    key: string;
    default?: boolean;
    label?: string;
    correspondingStudioInstanceUrl: string;
  }
```